modules/ROOT/pages/amazon-sns-connector.adoc

```diff
@@ -2,63 +2,57 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: anypoint studio, connector, sns, amazon sns
 
-*_Select_*
+Support Category: Select
 
-[[intro]]
-== Introduction
-The Anypoint Amazon SNS Connector provides connectivity to the Amazon Simple Notification Service (Amazon SNS) API, enabling you to build distributed web-enabled applications. Mule applications can use the Amazon SNS Connector to easily push real-time notification messages to interested subscribers over multiple delivery protocols.
+The Anypoint Connector for Amazon SNS provides connectivity to the Amazon Simple Notification Service (Amazon SNS) API, enabling you to build distributed web-enabled applications. Mule applications can use the Amazon SNS Connector to easily push real-time notification messages to interested subscribers over multiple delivery protocols.
 
-*Release Notes:* xref:release-notes::connector/amazon-sns-connector-release-notes.adoc[Amazon SNS Connector Release Notes]
-
-[[prerequisites]]
 == Prerequisites
+
 To be able to use the Amazon SNS Connector, you must have the following:
 
 * Access to Amazon Web Services - SNS.
-* To access AWS with the connector, you will need the credentials in the form of IAM.
+* AWS Identity and Access Management (IAM) credentials.
 * Anypoint Studio - Enterprise Edition.
 
-[[requirements]]
-=== Requirements
-
-For Hardware and software requirements, please visit https://www.mulesoft.com/lp/dl/mule-esb-enterprise[MuleSoft.com].
```

Comment:
Why was the link to Requirements removed?

Reply:
https://github.com/mulesoft/docs-mule-runtime/pull/362#discussion_r274791903